llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -915,9 +974,15 @@ void MemorySanitizer::createUserspaceApi(Module &M,
        AccessSizeIndex++) {
     unsigned AccessSize = 1 << AccessSizeIndex;
     std::string FunctionName = "__msan_maybe_warning_" + itostr(AccessSize);
+    SmallVector<Type *, 4> ArgsTy = {IRB.getIntNTy(AccessSize * 8),
```

Comment:
Should't this be SmallVector<Type *, 3>? I  see at most 3 things being in there

Reply:
I've shrunk it